tests/components/sabnzbd/test_config_flow.py

```diff
@@ -0,0 +1,113 @@
+"""Define tests for the Sabnzbd config flow."""
+from unittest.mock import patch
+
+from pysabnzbd import SabnzbdApiException
+
+from homeassistant import data_entry_flow
+from homeassistant.components.sabnzbd import DOMAIN
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PATH,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_URL,
+)
+
+from tests.common import MockConfigEntry
+
+VALID_CONFIG = {
+    CONF_NAME: "Sabnzbd",
+    CONF_API_KEY: "edc3eee7330e4fdda04489e3fbc283d0",
+    CONF_URL: "http://localhost:8080",
+    CONF_PATH: "",
+}
+
+VALID_CONFIG_OLD = {
+    CONF_NAME: "Sabnzbd",
+    CONF_API_KEY: "edc3eee7330e4fdda04489e3fbc283d0",
+    CONF_HOST: "localhost",
+    CONF_PORT: 8080,
+    CONF_PATH: "",
+    CONF_SSL: False,
+}
+
+
+async def test_create_entry(hass):
+    """Test that the user step works."""
+    with patch(
+        "homeassistant.components.sabnzbd.sab.SabnzbdApi.check_available",
+        return_value=True,
+    ):
+        result = await hass.config_entries.flow.async_init(
```

Comment:
Please patch the integration setup function for all create entry results to avoid setting up the whole integration. See the scaffold test template for example.